modules/assim.sequential/R/sda_plotting.R

```diff
@@ -918,20 +918,29 @@ post.analysis.multisite.ggplot <- function(settings, t, obs.times, obs.mean, obs
 ##' @param CI range of confidence interval.
 ##' @param unit list of unit used for y axis label.
 ##' @param style color option.
+##' @param PDF_w width of exported PDF file.
```

Comment:
What units? I'm guessing probably whatever `pdf()` defaults to?

Reply:
yes

Reply:
For cases like this I like to leave a breadcrumb trail:
```suggestion
##' @param PDF_w width of exported PDF file, passed on to `base::pdf()`.
```
(and similar for PDF_h)